website/content/docs/intro/index.mdx

```diff
@@ -49,71 +41,50 @@ Review the video below to learn more about Consul from HashiCorp's co-founder Ar
+
+Consul also has many integrations with Kubernetes that enable you to leverage Consul features in containerized environments. For example, Consul can automatically inject sidecar proxies into Kubernetes Pods and sync Kubernetes Services and non-Kubernetes services into the Consul service registry. 
+
+You can also schedule Consul workloads with [HashiCorp Nomad](https://www.nomadproject.io/) to provide secure service-to-service communication between Nomad jobs and task groups. 
+
+### Enable zero-trust network security
+
+Microservice architectures expose more of a network’s surface area, which demand additional security measures to prevent unwanted access by malicious actors. Consul provides several mechanisms that enhance network security without any changes to your application code, including mutual transport layer security (mTLS) encryption on all traffic between services and  Consul intentions, which are service-to-service permissions that you can manage through the Consul UI, API, and CLI.
+
+When you deploy Consul to Kubernetes clusters, you can also integrate with [HashiCorp Vault](http://vaultproject.io) to manage sensitive data. By default, Consul on Kubernetes leverages Kubernetes secrets as the backend system. Kubernetes secrets are base64 encoded, unencrypted, and lack lease or time-to-live properties. By leveraging Vault as a secrets backend for Consul on Kubernetes, you can manage and store Consul related secrets within a centralized Vault cluster to use across one or many Consul on Kubernetes datacenters. Refer to [Vault as the Secrets Backend](/docs/k8s/installation/vault) for additional information.
+
+You can also secure your Consul deployment, itself, by defining security policies in access control lists (ACL) to control access to data and Consul APIs.
+
+### Protect your services against network failure
+
+Outages are unavoidable, but with distributed systems it is critical that a power failure in one datacenter doesn’t disrupt downstream service operations. You can enable automated backups, redundancy zones, read-replicas, and other features that prevent data loss and downtime after a catastrophic event. L7 observability features also deliver service traffic metrics in the Consul UI, which help you understand the state of a service and its connections within the mesh.
+
+### Dynamically update network infrastructure devices
+
+Changes to your network, such as outdated firewall or load balancer rules, can lead to problems that disrupt operations at critical moments. You can deploy the Consul-Terraform-Sync (CTS) add-on to dynamically update network infrastructure devices when a service changes. CTS monitors the service information stored in Consul and automatically launches an instance of HashiCorp Terraform to drive relevant changes to the network infrastructure when Consul registers a change, reducing the manual effort of configuring network infrastructure.
```

Comment:
Suggestion: 

"One of the more difficult parts of the day-to-day operation of traditional network devices like firewalls and load balancers is simply updating lists of endpoints, and making sure those changes are done consistently and in a timely manner, instead of sitting in a ticket queue.

Consul-Terraform-Sync (CTS) can help solve this problem by leveraging the Consul service catalog, which is a list of all services and their endpoints, along with HashiCorp Terraform, the industry-standard infrastructure as code tool. The result: as information about service endpoints change in the Consul service catalog, Terraform is automatically triggered to push that new information to network devices. Automation means fewer mistakes and more rapid innovation by bypassing manual processes and ticket queues."